introduction.tex

```diff
@@ -1,2 +1,40 @@
+SymPy is an full featured computer algebra system (CAS) written in the Python
+programming language. It is open source, licensed under the extremely
+permissive BSD license, which allows anyone to reuse SymPy or its source code,
+even for commercial purposes. SymPy was started by Ond\v{r}ej \v{C}ert\'{\i}k
+in 2005, and it has since grown into a large open source project, with over
+500 contributors.
+% citation?
 
+SymPy is written entirely in the Python programming language,
+% cite Python?
+which is also the language used to interact with it, both programmatically and
+interactively. Python is a popular dynamically typed programming language that
+has a focus on ease of use and readability. Unlike many CASs, SymPy does not
+invent its own programming language. We rather take the approach that Python
+is already a well-designed programming language, built by expert language
+designers, something which the majority of SymPy developers are not. Python is
```

Comment:
Yeah, I'd kill the last couple of "clauses".  Rather I'd emphasize that this is about code reuse and reduction of effort and eases the learning curve for new developers. The value one places on language design is subjective.